news_items/028_java_grader/index.html

```diff
@@ -62,9 +62,4 @@
   To read more about new grader features you can check
   out <a href="https://prairielearn.readthedocs.io/en/latest/java-grader/">the
   documentation</a>.
-</p>
-
-<hr class="mt-5">
-<p class="text-right small">
-    Want to help make PrairieLearn better? It's open source and <a href="https://github.com/PrairieLearn/PrairieLearn">contributions are welcome!</a>
-</p>
+</p>
```

Comment:
There's a few files without trailing newlines here.

Reply:
Should be fixed by 845bfdc.